packages/imperative/src/imperative/__tests__/plugins/utilities/NpmFunctions.unit.test.ts

```diff
@@ -47,6 +48,24 @@ describe("NpmFunctions", () => {
         expect(spawnSyncSpy.mock.calls[0][1]).toEqual(expect.arrayContaining(["--registry", fakeRegistry]));
         expect(result).toBe(stdoutBuffer.toString());
     });
+    it("should write output to daemon stream if available", () => {
+        const writeMock = jest.fn().mockReturnValue("true");
+
+        jest.spyOn(ImperativeConfig, "instance", "get").mockReturnValue({
+            envVariablePrefix: "MOCK_PREFIX",
+            cliHome: "/mock/home",
+            daemonContext: {
+                stream: {
+                    write: writeMock
+                }
+            }
+        } as any);
+
+        jest.spyOn(ExecUtils, "spawnAndGetOutput").mockReturnValue(Buffer.from("Install Succeeded"));
+        const result = npmFunctions.installPackages("samplePlugin", { prefix: "fakePrefix" });
+        expect(writeMock).toHaveBeenCalledWith(DaemonRequest.create({ stderr: "Install Succeeded" }));
```

Comment:
Something along the lines of this assertion would be useful for the YargsConfigurer test (or just a check that write has been called)